src/screens/activity/PostItem.js

```diff
@@ -1,22 +1,100 @@
 import React from 'react';
-import { Text, Image } from 'react-native';
-import { Card } from 'react-native-paper';
+import { View, Text, StyleSheet, Image } from 'react-native';
+import Row from '../../components/Row';
+import ActionButton from '../../components/ActionButton';
+import { Avatar } from 'react-native-paper';
+import { getPostTime } from '../../utils/date';
 
-const PostItem = ({ feed }) => (
-  <Card style={{ margin: 8 }}>
-    <Card.Content>
-      <Text style={{ fontWeight: 'bold' }}>{feed.name}</Text>
-      <Text>{feed.content}</Text>
-      {feed.postImage && (
+const PostItem = ({ feed }) => {
+  const getNameInitials = name => {
+    var nameInitials = name.match(/\b(\w)/g).join(' ');
+    return nameInitials;
+  };
+  const {
+    profileImageUrl,
+    postImage,
+    content,
+    name,
+    likesCount,
+    commentsCount,
+    date,
+  } = feed;
+  return (
+    <View style={styles.rootContainer}>
+      <Row>
+        {profileImageUrl ? (
+          <Avatar.Image
+            size={36}
+            source={{
+              uri: profileImageUrl,
+            }}
+          />
+        ) : (
+          <Avatar.Text size={36} label={getNameInitials(name)} />
+        )}
+
+        <View style={{ paddingHorizontal: 5 }}>
+          <Text>{name}</Text>
+          <Text style={styles.dateText}>{getPostTime(date)}</Text>
+        </View>
+      </Row>
+      <Text style={styles.descriptionText}>{content}</Text>
+      {postImage && (
         <Image
-          source={{ uri: feed.postImage }}
-          style={{
-            height: 200,
+          style={styles.imageStyle}
+          source={{
+            uri: postImage,
           }}
         />
       )}
-    </Card.Content>
-  </Card>
-);
+      <Row style={styles.bottomRowContainer}>
+        {likesCount && <Text style={styles.countText}>5 Likes</Text>}
```

Comment:
likes and comments should be always visiable, if there are none, make 0 likes or 0 comments.